layouts/partials/search.html

```diff
@@ -0,0 +1,88 @@
+<link href="/pagefind/pagefind-ui.css" rel="stylesheet">
+<script src="/pagefind/pagefind-ui.js"></script>
```

Comment:
Where are these files?

Reply:
Those files get generated when `pagefind` runs, so will not be part of this project's source code.  (https://pagefind.app/docs/)